extensions/grpc/deployment/src/test/java/io/quarkus/grpc/server/services/AssertHelper.java

```diff
@@ -17,6 +20,16 @@ public static void assertRunOnEventLoop() {
         assertThat(Thread.currentThread().getName()).contains("eventloop");
     }
 
+    public static Context assertRunOnDuplicatedContext() {
+        assertThat(Vertx.currentContext()).isNotNull();
+        assertThat(isRootContext(Vertx.currentContext())).isFalse();
+        return Vertx.currentContext();
+    }
+
+    private static boolean isRootContext(Context context) {
+        return context instanceof EventLoopContext || context instanceof WorkerContext;
```

Comment:
Would it be good to have this method in some globally accessible utility class?

Reply:
We are working on that. It should land in a smallrye common module (need the same in various smallrye project)